src/js/components/Anchor/index.d.ts

```diff
@@ -29,7 +29,16 @@ export interface AnchorProps {
     | 'xlarge'
     | 'xxlarge'
     | string;
-  weight?: 'normal' | 'bold' | number;
+  weight?:
```

Comment:
We don't typically explicitly type for all possible browser CSS values and instead just use "string" for that case. That helps direct the user to using the typical grommet values while still allowing less frequently used ones.

However, if there's a way to leverage the default typing somehow via some indirection, like "these values plus any default values", that would be fine too. Not sure that we can do that though.